core/hosting/HostWithCoreClrHost/build.sh

```diff
@@ -0,0 +1,16 @@
+SCRIPTPATH=$(readlink -f "$0")
+BASEDIR=$(dirname $SCRIPTPATH)
+SRCDIR=${BASEDIR}/src
+OUTDIR=${BASEDIR}/bin/linux
+
+# Make output directory, if needed
+if [ ! -d "${OUTDIR}" ]; then
+    mkdir -p ${OUTDIR}
+fi
+
+# Build managed component
+echo Building Managed Library
+dotnet publish --self-contained -r linux-x64 ${SRCDIR}/ManagedLibrary/ManagedLibrary.csproj -o ${OUTDIR}
+
+# Build native component
+g++ -o ${OUTDIR}/SampleHost -D LINUX ${SRCDIR}/SampleHost.cpp -ldl
```

Comment:
This script is assuming Linux-x64.  `uname -m` will return the actual machine type. It is not a direct mapping though..

Reply:
Yea, I'll just make a note that this builds for Linux-x64 and that people can change the dotnet publish line, if needed.